docs/metadata_etl.rst

```diff
@@ -0,0 +1,198 @@
+.. _metadata-etl:
```

Comment:
This is copied verbatim from the current docs. I thought there was too much material to go with `unpackaged`.

Reply:
I like positioning it right after `unpackaged` and making sure that both topics contextualize themselves in relation to the other. They are two different approaches to the same problem space of post-install configuration.